R/boot.R

```diff
@@ -69,8 +69,13 @@ setMethod("parboot", "unmarkedFit",  function(object, statistic=SSE, nsim=10,
     simdata <- replaceY(object@data, x)
     tryCatch({
       #if(runif(1,0,1) < 0.5) stop("fail") # for testing error trapping
-      fit <- update(object, data=simdata, starts=starts, se=FALSE)
-      statistic(fit, ...)
+      if (class(object) == "unmarkedFitCOP") {
```

Comment:
I changed it back: https://github.com/rbchan/unmarked/pull/267/commits/94ab036c847ce1ae47627853042e7846206848ac

And added the `starts` argument to `occuCOP`:  https://github.com/rbchan/unmarked/pull/267/commits/6484766345ebd17c60c214917f0c86c4af569765#diff-7edadca97caca7505bcd2f68d15a7f43293f08eb46c0f72c446fecb767f6f158R617 . I kept the `psistarts` and `lambdastarts` arguments as well because it seemed to me that it was simpler to understand with separated formulas.